aptos-move/aptos-transactional-test-harness/src/aptos_test_harness.rs

```diff
@@ -304,19 +304,42 @@ static PRECOMPILED_APTOS_FRAMEWORK_V2: Lazy<PrecompiledFilesModules> = Lazy::new
         .map(|(string, num_addr)| format!("{}={}", string, num_addr))
         .collect();
 
-    let options = move_compiler_v2::Options {
-        sources: aptos_cached_packages::head_release_bundle()
+    // aptos-experimental can be using latest language features, while others cannot.
+    // So we need to compile it twice, once without aptos-experimental, and once with.
+
+    let (experimental_sources, mut prod_sources): (Vec<String>, Vec<String>) =
+        aptos_cached_packages::head_release_bundle()
             .files()
-            .unwrap(),
+            .unwrap()
+            .into_iter()
+            .partition(|f| {
+                f.contains("aptos-move/framework/aptos-experimental/sources")
```

Comment:
I don't see a cleaner way to separate aptos-experimental at this point in the code